arches/app/media/js/views/components/datatypes/concept.js

```diff
@@ -12,6 +12,11 @@ define(['arches', 'knockout', 'viewmodels/concept-select'], function (arches, ko
                 this.multiple = ko.observable(false);
                 this.searchValue = ko.observable(filter.val || '');
                 this.node = params.node;
+                this.clearSelectionEventHandler = function(data,event){
```

Comment:
Hi @aarongundel - I've moved the select2Config.clear() from the template into the view model so it'll work for a click or a keyup when the enter key is pressed.

Are you happy with this pattern and the name of the function? If so, I'll apply the same pattern for all the other instances of this.

Reply:
@csmith-he - yep that'll work nicely!  Feel free to apply that to all the instances of this issue.